tests/tools/CMakeLists.txt

```diff
@@ -2,6 +2,9 @@
 # Licensed under the MIT License.
 
 if (OE_SGX)
+  if (UNIX)
+    add_subdirectory(oeapkman)
```

Comment:
why only for SGX?  I expected 5-7 to be before line 4.

Reply:
Currently, oeapkman is SGX Linux only.

It does not yet support  Windows because, it would need WSL or docker and a powershell wrapper to provide equivalent functionality as Linux implementation. This definitely is possible; but I'm not sure what it would take to lock down WSL/docker in CI.

It does not yet support OPTEE since the enclave development arch (x86_64) is different from the actual enclave arch (ARM/AARCH64). It is possible to support most of the functionality using binfmt-misc (qemu user-space emulation) or by running within qemu.

Reply:
The PR description says nothing about being SGX only.  If it is, then update the PR title and description.

Reply:
Done.